src/api/CMakeLists.txt

```diff
@@ -40,29 +40,26 @@ PUBLIC ${PUBLIC_SOURCES_TO_ADD}
 
 set(OBJECTS_TO_ADD $<TARGET_OBJECTS:api> ${OBJECTS_TO_ADD})
 
-if (NOT USE_READLINE)
-  set(line_library ${LibEdit_LIBRARIES})
-else()
-  set(line_library ${Readline_LIBRARY})
-endif()
-
 if (BUILD_SHARED_LIBS)
   add_library(api_shared SHARED ${OBJECTS_TO_ADD})
```

Comment:
Can we exclude Interpret.{cc,h} here?

Reply:
As agreed, we move that change to separate PR